test/lib/utils/helper.ts

```diff
@@ -440,3 +440,61 @@ export const forkSetup = async (
         await approveToken(client, accountAddress, tokens[i], vaultVersion);
     }
 };
+
+/**
+ * Setup local fork with approved token balance for a given account address
+ *
+ * @param client Client that will perform transactions
+ * @param accountAddress Account address that will have token balance set and approved
+ * @param tokens Token addresses which balance will be set and approved
+ * @param slots Slot that stores token balance in memory - use npm package `slot20` to identify which slot to provide
+ * @param balances Balances in EVM amounts
+ * @param jsonRpcUrl Url with remote node to be forked locally
+ * @param isVyperMapping Whether the storage uses Vyper or Solidity mapping
+ */
+export const forkSetupCowAmm = async (
+    client: Client & PublicActions & TestActions & WalletActions,
+    accountAddress: Address,
+    tokens: Address[],
+    slots: number[] | undefined,
+    balances: bigint[],
+    spender: Address,
```

Comment:
Because this is specifically for CowAMM lets rename spender to pool as it makes it a bit clearer to follow when reading the function alone?

Also some of the doc comments are out of date.

Reply:
I started it with `pool` and later changed to `spender`, but I agree that pool is easier to read 👍

Reply:
done ✅